src/RemoteAPI/class-validate-api.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Class for Validate API (`/validate`).
```

Comment:
```suggestion
 * Class for Validate API (`/validate`)
```
Really super minor detail, but [file headers don't finish with a period](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers). Strange but true! 🙂

Reply:
Interesting! I adapted from the structure of the [ `class-related-api.php` ](https://github.com/Parsely/wp-parsely/blob/develop/src/RemoteAPI/class-related-api.php#L3) class and it also had the period.  :smile: Fixing it.

Reply:
Yes, some files that have a period haven't been updated. You happened to stumble on one of those it seems 🙂